squad/frontend/templatetags/squad.py

```diff
@@ -150,7 +150,10 @@ def download_build_attachments_url(group_slug, project_slug, build_version, test
 @register_global_function
 def project_status(project):
     if project.latest_build is not None:
-        return project.latest_build.status
+        try:
+            return project.latest_build.status
+        except Build.status.RelatedObjectDoesNotExist:
+            return None
```

Comment:
Minor, but when in a loop you can just "return" without None.

Reply:
I think `return None` is clearer in this case since the other return statement returns something (this also aligns with PEP8's "be consistent in return statements" recommendation https://peps.python.org/pep-0008/#programming-recommendations)